src/inputSources/mouseInput.js

```diff
@@ -89,6 +89,12 @@ function mouseDown (e) {
     type: eventType
   };
 
+  const onMouseDown = $(element).data('onMouseDown');
```

Comment:
This seems like a weird way to add a function to run on an element on mouse down. What are you using this for? Why shouldn't users just add an event handler to CornerstoneToolsMouseDown instead?